lib/DDG/Spice/Symbolab.pm

```diff
@@ -12,12 +12,20 @@ code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/
 spice to => 'https://www.symbolab.com/ddg?query=$1';
 spice wrap_jsonp_callback => 1;
 
-triggers startend => "calculate", "solve", "compute";
-handle remainder => sub {
-    if ($_ =~ /[\^\*\=\+\-\/\\]/ or $_ =~ /\b(sin|cos|tan|cot|csc|sec|ln|log|sqrt|integral)/i) {
-        return $_;
+triggers query_lc => qr/\>|\<|\+|\^|\*|\=|\\|\/|\b(tan|sin|cos|csc|cot|sec|ln|log|sqrt|derivative|integral|limit)\b/;
+
+handle query_lc => sub {
+    my $query = $_;
+    
+    $query =~ s/(calculate|solve|compute) ?//;
+    
+    # checks if its a simple case that only has numbers, and math operations, in which case it assumes 
+    # DDG calculator goodie should handle this and returns
+    if ($query =~ /^(\d| |\+|\-|\*|\^|\/|sqrt|sin|cos|tan|cot|ln|log|sec|csc|\(|\))+$/){
+        return;
     }
-    return;
+    
+    return $query;
```

Comment:
I think this is going to be too broad -- we need to ensure that we at least have have numbers to deal with.

For example, I `<span id="test">` would trigger this. Likewise a URL would also trigger this as well: `http://test.com`

![1____projects_zeroclickinfo-spice__ssh_](https://cloud.githubusercontent.com/assets/873785/11346510/04f86098-91eb-11e5-9549-6fccd37aec00.png)

We may need to investigate a math parser lib like https://metacpan.org/pod/Math::Expr or https://metacpan.org/pod/Math::Symbolic to ensure the expression is valid.

I'll speak to the team and see what they think is the best approach here...